### ISSUE TITLE
Accuracy after epoch is not shown

### ISSUE THREAD

**Issue by AhmedBahaaElDinMohammed** [kaushaltrivedi/fast-bert#79]
I am using the same exact code as the example_notebook but with different dataset and only loss is shown in the very end of training , no accuracy is shown after the validation step. what should i do ?

**Comment by arrrrrmin** [comment]
I'm having the same problem. 
`"do_train": True,
    "do_eval": True,
    "do_lower_case": True,
    ....
    "eval_all_checkpoints": True,
`
But during the training time it's showing me the progress bar, but no validation per step.
The validation output seems to be overwritten by this progress bar in the shell (just an idea).
Greetings Armin

**Comment by arrrrrmin** [comment]
Ok, sorry just figured it out. I accidently disabled the logger 

`python
import logging

logfile = str(LOG_PATH/'log-{}-{}.txt'.format(run_start_time, args["run_text"]))

logging.basicConfig(
    level=logging.INFO,
    format='%(asctime)s - %(levelname)s - %(name)s -   %(message)s',
    datefmt='%m/%d/%Y %H:%M:%S',
    handlers=[
        logging.FileHandler(logfile),
        logging.StreamHandler(sys.stdout)
    ])

logger = logging.getLogger()
`
@AhmedBahaaElDinMohammed maybe you got a non existing `log_path` or `log_file`?

**Comment by DecentMakeover** [comment]
@arrrrrmin  Which function takes care of generating eval scores during training, i want to print additional metrics along with accuracy.Thanks.

**Comment by arrrrrmin** [comment]
@DecentMakeover There is an extra module for that. See: https://github.com/kaushaltrivedi/fast-bert/blob/master/sample_notebooks/new-toxic-multilabel.ipynb for examples.

`from fast_bert.metrics import accuracy_multilabel, accuracy_thresh, fbeta, roc_auc
...
metrics = []
metrics.append({'name': 'accuracy_thresh', 'function': accuracy_thresh})
metrics.append({'name': 'roc_auc', 'function': roc_auc})
metrics.append({'name': 'fbeta', 'function': fbeta})
...

learner.fit(args.num_train_epochs, args.learning_rate, validate=True)
`
Make sure to set `validate` to `True`, otherwise you won't see your metrics during Training.
